frontend/src/features/drug_discovery/components/targets/TargetDetail.vue

```diff
@@ -285,12 +285,7 @@ export default {
     this.target.data.pdb_file = new File([new Blob([this.target.data.pdbContents])], "protein.pdb");
     this.quasar = useQuasar();
```

Comment:
Don't need this anymore. Global quasar is available through this.$q if you use defineComponent